tests/base/generate_normal_random_number_01.cc

```diff
@@ -54,11 +54,11 @@ main()
   initlog();
 
   // create 4 threads, run the test function on them
-  Threads::Thread<std::pair<double, double>> tg[4];
-  tg[0] = Threads::new_thread(&test);
-  tg[1] = Threads::new_thread(&test);
-  tg[2] = Threads::new_thread(&test);
-  tg[3] = Threads::new_thread(&test);
+  std::async tg[4] = {};
```

Comment:
Wait, `std::async` is a function. How does this work? Did you mean this to be
```suggestion
  std::vector<std::future> tg;
```
?

Reply:
Fixed now.